### ISSUE TITLE
Adding the conditional batch normalization & projection discriminator

### ISSUE THREAD

**Issue by samanehAb** [taki0112/Self-Attention-GAN-Tensorflow#18]
Are you planning to add the conditional batch normalization and projection discriminator similar to cGANs with Projection Discriminator (https://arxiv.org/abs/1802.05637)?

**Comment by China-LiuXiaopeng** [comment]
You can refer to the tf implementation of conditional batch normalization here https://github.com/MingtaoGuo/sngan_projection_TensorFlow

**Comment by ahfares** [comment]
+1, are you planning to follow the paper description and adding the conditional batch normalization and projection discriminator?
